tests/test_builders/test_build_linkcheck.py

```diff
@@ -436,7 +438,7 @@ def do_GET(self):
     assert row['status'] == 'ignored'
 
 
-def custom_handler(valid_credentials=(), success_criteria=lambda _: True):
+def custom_handler(valid_credentials: tuple[str, str] | None = None, success_criteria: Callable[[Any], bool] = lambda _: True) -> type[BaseHTTPRequestHandler]:
```

Comment:
Split the signature.

Reply:
what do you mean?

Reply:
I meant to be on two lines (but maybe it's not that long, it might be the GitHub UI that made me felt that way).